### PR TITLE
Scenario listener test

### PR DESCRIPTION
**Pull request by jurok3x**

### CODE REVIEW

**Review by Sokol3783 — APPROVED** [review]

File: src/main/java/executor/service/handler/TerminatorListener.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+package executor.service.handler;
+
+public interface TerminatorListener {
```
Comment (OlexTul) [inline src/main/java/executor/service/handler/TerminatorListener.java:3]: Explain why you are coding this code and trying to make changes to the already killed code in the main branch (it looks like a stub). If you need this for tests, to change the logic of the while loop, then you can use Reflection (**java.lang.reflect**). An example of this approach can be seen in ParallelFlowExecutorServiceImplTest [#32](https://github.com/GFLCourses5/worker/pull/32). Thanks

**Review by OlexTul — CHANGES_REQUESTED** [review]

File: src/main/java/executor/service/handler/TerminatorListener.java (hunk 1 above)
Comment (jurok3x) [reply src/main/java/executor/service/handler/TerminatorListener.java:3]: Not sure what do you mean under "trying to make changes to the already killed code in the main branch". Please provide me more clear and detailed explanation.

Comment (OlexTul) [reply src/main/java/executor/service/handler/TerminatorListener.java:3]: Sorry, into already approved code, beta-main branch.

Comment (jurok3x) [reply src/main/java/executor/service/handler/TerminatorListener.java:3]: The proposed code does not change *ScenarioListener* logic. The only purpose is to perform testing. And I don't want to put additional flags and use reflection.

**Review by OlexTul — APPROVED** [review]

**Review by IfLunatic — APPROVED** [review]

**Pull request merged**